data/drak/drak.txt

```diff
@@ -234,6 +234,8 @@ ship "Archon"
 	turret 55 16 "Drak Distancer"
 	explode "nuke explosion" 3
 	"final explode" "final explosion large"
+	description "Very little is know about Archons other than that they significantly exceed the spacefaring prowess of any known species. From external observation, they appear to be a mix of organic and cybernetic elements. They are known to guard noteworthy locations and sapient species or the remains thereof."
```

Comment:
```suggestion
	description "Very little is known about Archons other than that they significantly exceed the spacefaring prowess of any known species. From external observation, they appear to be a mix of organic and cybernetic elements. They are known to guard noteworthy locations and sapient species - or the remains thereof."
```
I like the dash because I think the sentence sounds better with a pause there, but it's personal preferenece.
Also, [sapient](https://www.merriam-webster.com/dictionary/sapient) seems like odd word choice for that line, unless there's another definition I don't know about.